### PR TITLE
attempt transform test for positional param

### PR DESCRIPTION
**Pull request by NullVoxPopuli**

### CODE REVIEW

File: tests/ember-template-recast-test.js [hunk 1]
```diff
@@ -15,6 +15,46 @@ function run(args, cwd) {
   });
 }
 
+QUnit.module('ember-template-recast adding "this" to properties in PathExpression', function(hooks) {
```
Comment (Turbo87) [inline tests/ember-template-recast-test.js]: can you remove this change?

Comment (NullVoxPopuli) [reply tests/ember-template-recast-test.js]: done!

File: tests/ember-template-recast-test.js [hunk 2]
```diff
@@ -15,6 +15,48 @@ function run(args, cwd) {
   });
 }
 
+QUnit.module('ember-template-recast adding "this" to properties in PathExpression', function(
```
Comment (rwjblue) [inline tests/ember-template-recast-test.js]: Can you move this yo `index-test.js`?

Comment (rwjblue) [reply tests/ember-template-recast-test.js]: This file is specifically for testing the binary

Comment (rwjblue) [reply tests/ember-template-recast-test.js]: Actually, I see you added the other test, mybe just remove this one?

Comment (NullVoxPopuli) [reply tests/ember-template-recast-test.js]: done! I thought I already did this, but didn't push. ha

**Comment by rwjblue** [comment]
In order to fix this, we need to dig in to find out where the issue is introduced. Can you throw some console.logs around here:

https://github.com/ember-template-lint/ember-template-recast/blob/f7f3f3c5fca83ebd10fa87a75d5e5131fb2fd9a0/src/index.js#L202

We need to know if that updatedLine is correct, and if not what’s wrong with it (either the inputs are wrong or the calculation is wrong) that will tell us where to dig in to.

File: tests/index-test.js [hunk 3]
```diff
@@ -453,6 +453,39 @@ QUnit.module('whitespace and removed hash pairs', function() {
...
+      let { builders: b } = env.syntax;
+      return {
+        PathExpression(ast) {
+          let token = ast.original;
+
+          if (token === 'positional') {
+            return b.path(`this.${token}`);
+          }
+
+          return ast;
```
Comment (Turbo87) [inline tests/index-test.js]: I think the `return ast` is not necessarily needed as it's the default when the visitor function does not return anything

Comment (NullVoxPopuli) [reply tests/index-test.js]: fixed

**Comment by rwjblue** [comment]
Thanks for the test!

**Pull request merged**